telegram_bot/bot_handlers/handle_view_statistic.py

```diff
@@ -0,0 +1,26 @@
+from database.users_quiz_progress_table import fetch_user_stat_by_topic
+from telegram_bot.bot_message_generators.statistic_menu_generator import send_user_statistic_menu
+
+
+async def handle_view_statistic_menu(call, bot, chat_id):
```

Comment:
Why function name is "handle_view_statistic_menu"? You create menu or send user information about his achievements?
Rename it based on its ancestor (callback_function from quiz_bot.py )

Reply:
So why are you passing the chat_id to the handler if you can get it by call.message.chat.id?